src/App.js

```diff
@@ -1,10 +1,53 @@
-import logo from './logo.svg';
 import './App.css';
+import React from 'react';
+import allRecipes from './recipes.js';
+
+const allRecipes_keys = Object.keys(allRecipes);
+
+class Header extends React.Component {
+  render () {
+    return (
+      <div>
+        <h1>My Recipes</h1>
+      </div>
+    );
+  }
+}
+
+function RecipeItem(props) {
+  return (
+    <li>
+      <p className="text-gray-500">{props.recipe.name}</p>
+    </li>
```

Comment:
What do you think about dropping the `<li>` surrounding tag? This would make the component more flexible for rendering anywhere.

You could move the `<li>` inside of the `map` you do when defining `listItems` below.

Reply:
This entire function will be changed based on new designs. Just a placeholder to ensure things are working fine and the content will be rendered